.github/workflows/vale_tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Linting
+on: [push]
+jobs:
+  prose:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: '[CHECKOUT] Checkout branch that triggered this workflow'
+      uses: actions/checkout@v2
+
+    - name: 'Set up Ruby 2.6'
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+
+    - name: 'Install Asciidoctor'
+      run: gem install asciidoctor
+
+    - name: Set up Homebrew
+      id: set-up-homebrew
+      uses: Homebrew/actions/setup-homebrew@master
+
+    - name: Run vale for docs linting
+      run: |
+        brew install vale
+        vale --debug modules/
```

Comment:
Instead of using the available [vale action](https://github.com/errata-ai/vale-action#usage), you are installing `vale` via `homebrew` and calling it manually. What is the problem with using the vale action?

Reply:
I tested the action several ways. The action was not picking up styles and therefor throwing a zero results as nothing was being compared "as we covered yesterday". I checked the Homebrew repo as an example and tried using their method. This way works though not as the action intends. 

Of course I agree with you. We should swap back to the default setup. Just wrapping my head around whats going on.

Reply:
I am now using the default methods.